File: 01 - Multiples of 3 or 5/cpp/main.cpp
#include <iostream>

int multiplesOf3and5(int limit)
{
    int sum = 0;
    for (int i = 1; i < limit; i++)
    {
        if (i % 3 == 0 || i % 5 == 0)
        {
            std::cout << i << std::endl;
            sum += i;
        }
    }
    return sum;
}

int main()
{
    int result = multiplesOf3and5(1000);
    std::cout << "result: " << result << std::endl;
}

File: 01 - Multiples of 3 or 5/python/main.py
def multiplesOf3and5(limit):
    sum = 0
    for i in range(1, limit):
        if(i % 3 == 0 or i % 5 == 0):
            print(i)
            sum += i
    return sum

result = multiplesOf3and5(1000)
print("result:" + str(result))